docs/add_dataset.md

```diff
@@ -607,6 +630,9 @@ class MyDatasetTest(tfds_test.DatasetBuilderTestCase):
       "name1": "path/to/file1",  # Relative to fake_examples/my_dataset dir.
       "name2": "file2",
   }
+  # If you want to change fake_data path:
```

Comment:
I would remove that part here.
That point is already touched in the other section.